src/main/java/uk/gov/hmcts/reform/sandl/snlevents/repository/queries/HearingQueries.java

```diff
@@ -271,7 +271,7 @@ private Predicate createEqualsPredicate(CriteriaBuilder cb, Root<Hearing> hearin
         Root<Person> subQueryPersonRoot = subQueryPerson.from(Person.class);
         subQueryPerson.where(
             //join subquery with main query
-            cb.equal(subQueryPersonRoot.get("id"), hearingRoot)
+            cb.equal(subQueryPersonRoot.get("id"), hearingRoot.get(Hearing_.reservedJudge))
```

Comment:
I think we can change this hardcode string into selector. Can you try do it within your fix?

Reply:
Done :)